docs/data-sources/cbh_instances.md

```diff
@@ -54,24 +52,14 @@ In addition to all arguments above, the following attributes are exported:
 <a name="CbhInstances_Instance"></a>
 The `Instance` block supports:
 
-* `publicip_id` - Indicates the ID of the elastic IP.
-
-* `exp_time` - Indicates the expire time of the instance.
-
-* `start_time` - Indicates the start time of the instance.
-
-* `end_time` - Indicates the end time of the instance.
+* `public_ip_id` - Indicates the ID of the elastic IP.
 
-* `release_time` - Indicates the release time of the instance.
+* `public_ip` - Indicates the elastic IP address.
 
 * `name` - Indicates the instance name.
```

Comment:
The fields `name`、`vpc_id`、`subnet_id`、`security_group_id`、`flavor_id` and `version` has already defiend in `Argument Reference`.